CHANGELOG.md

```diff
@@ -12,8 +12,9 @@ To install release candidates run `[sudo] gem install cocoapods --pre`
 
 ##### Bug Fixes
 
-* None.  
-
+* Fix linking of vendored libraries and frameworks in pod targets
```

Comment:
nit please add 2 empty spaces in the end of this line for formatting.

Reply:
Oh, right. Done.